### ISSUE TITLE
Pharmacy preset should have dispensing field

### ISSUE THREAD

**Issue by eehpcm** [openstreetmap/iD#4763]
Most pharmacies will be dispensing=yes, although OTC-only pharmacies will be dispensing=no.  This can be added by knowledgeable users under "All tags" but it would be helpful to newbies (and make life a tad easier for knowledgeable users) if this were prompted under "All fields."  More useful than offering "Drive-Through" because I've never seen a drive-through pharmacy here in the UK but I've yet to see a pharmacy that wasn't also dispensing=yes.

**Comment by bhousel** [comment]
> I've yet to see a pharmacy that wasn't also dispensing=yes.

If that's the case, then why tag it at all?

**Comment by eehpcm** [comment]
Five reasons.

1) Although I've never seen a pharmacy in the UK that wasn't dispensing, I haven't seen all UK pharmacies.  There may be some that aren't.

2) The situation may be very different in other countries.  This is implied by the fact that the wiki (down, as I write this) has a page for the dispensing tag which says (google cache is helpful here) for dispensing=no "A pharmacy with a pharmacist that does not fill prescriptions, but does sell pharmacist-only drugs (e.g., pseudoephedrine in the US)."

3) There is no indication in the wiki what value is assumed for this tag if it is not present.  Any of "yes," "no," or "unknown" might be assumed.  And might be assumed differently by different renderers.

4) I doubt there are many drive-through pharmacies in the UK.  Let's remove that too, irrespective of how many there may be in the US. :)

5) I have a vague memory from the last pharmacy I mapped that the renderer didn't show a pharmacy icon unless dispensing=yes was present.  This isn't tagging for the renderer, that pharmacy was a dispensing pharmacy.  But I wouldn't have thought to add that tag until I started investigating why the icon wasn't showing.

But if you don't want to, you don't want to.
